### PR TITLE
Updated modify_yaml with docstring and clarifications

### PR DESCRIPTION
**Pull request by ashcrow**
Nothing earth shattering.

### CODE REVIEW

File: library/modify_yaml.py [hunk 1]
```diff
@@ -21,8 +26,18 @@
 '''
 
 
-# pylint: disable=missing-docstring
 def set_key(yaml_data, yaml_key, yaml_value):
+    ''' Updates a parsed yaml structure setting a key to a value.
```
Comment (tbielawa) [inline library/modify_yaml.py]: I <3 documentation.

File: library/modify_yaml.py [hunk 2]
```diff
@@ -95,14 +111,9 @@ def none_representer(dumper, data):
 
     # ignore broad-except error to avoid stack trace to ansible user
     # pylint: disable=broad-except
-    except Exception as e:
-        return module.fail_json(msg=str(e))
-
+    except Exception as error:
+        return module.fail_json(msg=str(error))
```
Comment (tbielawa) [inline library/modify_yaml.py]: @ashcrow just a cosmetic change here?

Comment (ashcrow) [inline library/modify_yaml.py]: That one is, yes.

**Comment by ashcrow** [comment]
aos-ci-test

**Comment by abutcher** [comment]
Heads up that [yedit](https://github.com/openshift/openshift-ansible/blob/master/roles/lib_utils/library/yedit.py) will replace this module once usage in openshift-ansible has been swapped over.

**Pull request merged**